src/features/auth/signup/QuitSignupModal.tsx

```diff
@@ -16,14 +17,21 @@ import { ColorsEnum, Typo } from 'ui/theme'
 
 interface Props {
   visible: boolean
+  signupStep: string
```

Comment:
Possible de typer les valeurs possibles ?
```suggestion
  signupStep: 'CGU' | 'Machin1' | etc...
```